From d3bb6df5353533686bc718ea96ff9b29d7f6c86c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alex=20R=C3=B8nne=20Petersen?= <alex@alexrp.com>
Date: Fri, 31 Oct 2025 15:45:01 +0100
Subject: [PATCH 031/133] std.Target: bump fuchsia max version to 28.0.0
X-Developer-Signature: v=1; a=openpgp-sha256; l=728; i=xtex@astrafall.org;
 h=from:subject; bh=oIoqjqk+CrM92oeQpnSvtaXJGzjAzJZdxv/JDy4+M90=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn7cv/mJOlOM5M2KSlZp1XB1V59KIFfZ3MD55zkI28lA
 6/ez5vaMZGFQYyLwVJMkaXIsMGbVSedX3RZuSzMHFYmkCHSIg0MDAwMLAx8uYl5pUY6Rnqm2oZ6
 hoY6RjpGDFycAjDVe2Yx/DN4yprzcWfe6rh0QR3GN3y7Glp8vzA/PnBlz2ThrC9uW14zMkxiNt1
 3VnJypOJGTwdNBye/fueJ7PpcL8pfRekwH1R0ZAMA
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit 0116d98093df6d9c53877f4c449c9365e64b0f6e)
---
 lib/std/Target.zig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/std/Target.zig b/lib/std/Target.zig
index 9d3263732d19..2b56408e506e 100644
--- a/lib/std/Target.zig
+++ b/lib/std/Target.zig
@@ -406,7 +406,7 @@ pub const Os = struct {
                 .fuchsia => .{
                     .semver = .{
                         .min = .{ .major = 1, .minor = 0, .patch = 0 },
-                        .max = .{ .major = 27, .minor = 0, .patch = 0 },
+                        .max = .{ .major = 28, .minor = 0, .patch = 0 },
                     },
                 },
                 .hermit => .{
-- 
2.52.0

